### ISSUE TITLE
Embedded certificate bundle should include all Mozilla CA certs, otherwise CLI does not work on Windows

### ISSUE THREAD

**Issue by joshrosso** [vmware-tanzu/tanzu-framework#1710]
**Bug description**

_This originated from user feedback we've received in TCE (https://github.com/vmware-tanzu/community-edition/issues/1398)_

Built the `management-cluster` plugin from `0.16.0` and attempted to bootstrap a `managed-cluster`:

![image](https://user-images.githubusercontent.com/6200057/154384486-d59ab4ea-fc2a-4afc-a0e1-baee8e879a6d.png)

You can workaround this using https://github.com/vmware-tanzu/community-edition/issues/1398#issuecomment-902863369. But it should pull without this workaround.

Note that unmanaged cluster pulls a TKR and does **not** have this issue.

![image](https://user-images.githubusercontent.com/6200057/154384611-239ba1ac-6281-4238-96fd-a64f10684522.png)

**Affected product area (please put an X in all that apply)**

- [ ] APIs
- [ ] Addons
- [x] CLI
- [ ] Docs
- [ ] IAM
- [ ] Installation
- [ ] Plugin
- [ ] Security
- [ ] Test and Release
- [ ] User Experience

**Comment by randomvariable** [comment]
Via @stuartpreston :

> Ok, I think I have this figured out and it's pretty bad if you're a Windows user :(
TLDR: We are [embedding in the core CLI](https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/cli/clientconfighelpers/windows_trusted_ca.go) the root CA cert for [projects.registry.vmware.com](http://projects.registry.vmware.com/) - however the cert is the Root CA for the internal VIP and not the public access via CDN.

> More: [Tanzu CLI writes the embedded root CA cert file to the home directory on Windows and then adds it to the cert chain](https://github.com/vmware-tanzu/tanzu-framework/blob/fe23cf4d8a1d33251ebf2599b408419122a34808/pkg/v1/cli/clientconfighelpers/helpers.go#L67), which is why no-one using the CLI on Windows other than internal to VMware can download any plugins or do a tanzu init.

> Though the PR will need some consideration because any internal tests may break if they are run on a system that resolves [projects.registry.vmware.com](http://projects.registry.vmware.com/) and [projects-stg.registrty.vmware.com](http://projects-stg.registrty.vmware.com/) to an internal VIP.

**Comment by randomvariable** [comment]
The embedded CA bundle should be those of https://wiki.mozilla.org/CA/Included_Certificates 

We could add an at build time function for folk to add the VMware CA, but I don't think the general public needs to be exposed to that.
